### ISSUE TITLE
Package verification issue (shasum check) when installing vue/cli

### ISSUE THREAD

**Issue by mentiflectax** [npm/npm#21050]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [X] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] npm is producing incorrect or undesirable behavior.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

Error while trying to install [vue/cli](https://vuejs.org/v2/guide/installation.html).

#### How can the CLI team reproduce the problem?

I ran `npm install -g @vue/cli` on a Windows 7 machine. It generated the following output:

```
C:\Users\1>npm install -g @vue/cli
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "@vue/cli"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3

npm ERR! shasum check failed for C:\Users\1\AppData\Local\Temp\npm-2032-e40197a0
\registry.npmjs.org\_40types\accepts\-\accepts-1.3.5.tgz
npm ERR! Expected: c34bec115cfc746e04fe5a059df4ce7e7b391575
npm ERR! Actual:   a794a4c39dff6afd60aee3ddb42b43bf2a967a3e
npm ERR! From:     https://registry.npmjs.org/@types/accepts/-/accepts-1.3.5.tgz

npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\1\npm-debug.log
```
The requested [npm-debug.log](https://github.com/npm/npm/files/2127880/npm-debug.log) file is attached.

### supporting information:

 - `npm -v` prints: `3.10.3`
 - `node -v` prints: `v6.4.0`
 - `npm config get registry` prints: `https://registry.npmjs.org/`
 - Windows, OS X/macOS, or Linux?: Windows 7

**Comment by legodude17** [comment]
Does this occur on the latest version of npm?

**Comment by mentiflectax** [comment]
No. It disappeared after I upgraded npm. 22.06.2018, 17:19, "Joshua" <notifications@github.com>:Does this occur on the latest version of npm?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by legodude17** [comment]
Great. This can be closed then.
